services/arrow-flight/src/test/java/com/dremio/service/flight/AbstractTestFlightSqlServer.java

```diff
@@ -56,4 +56,51 @@ public void testGetCatalogs() {
       throw new RuntimeException(e);
     }
   }
+
+  @Test
+  public void testGetSchemasWithNoFilter() {
+    FlightSqlClient flightSqlClient = getFlightClientWrapper().getSqlClient();
+    FlightInfo flightInfo = flightSqlClient.getSchemas(null, null, getCallOptions());
+    try (FlightStream stream = flightSqlClient.getStream(flightInfo.getEndpoints().get(0).getTicket(), getCallOptions())) {
+      Assert.assertTrue(stream.next());
+      VectorSchemaRoot root = stream.getRoot();
+
+      final String[] expectedSchemas =
+        {"INFORMATION_SCHEMA", "cp", "dacfs", "dfs", "dfs_root", "dfs_test", "sys", "sys.cache"};
+
+      Assert.assertEquals(expectedSchemas.length, root.getRowCount());
+
+      VarCharVector catalogNameVector = (VarCharVector) root.getVector("catalog_name");
+      VarCharVector schemaNameVector = (VarCharVector) root.getVector("schema_name");
+
+      for (int i = 0; i < expectedSchemas.length; i++) {
+        String catalogName = catalogNameVector.getObject(i).toString();
+        String schemaName = schemaNameVector.getObject(i).toString();
+        Assert.assertEquals("DREMIO", catalogName);
+        Assert.assertEquals(expectedSchemas[i], schemaName);
+      }
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
Instead of catching and rethrowing as a RuntimeException, just change the test method to add throws Exception

Reply:
Fixed